Pipfile

```diff
@@ -4,31 +4,34 @@ verify_ssl = true
 name = "pypi"
 
 [dev-packages]
-mypy = "*"
-pytest = "*"
+mypy = "==0.620"
+pytest = "==3.7.3"
+
 # DO NOT UPGRADE, latest version breaks paths on Windows:
 # https://github.com/pytest-dev/pytest-django/issues/641
 pytest-django = "==3.3.3"
-"flake8" = "*"
-pytest-cov = "*"
-python-dotenv = "*"
-factory-boy = "*"
-requests-mock = "*"
-django-webtest = "*"
+
+"flake8" = "==3.5.0"
+pytest-cov = "==2.5.1"
+python-dotenv = "==0.9.1"
+factory-boy = "==2.11.1"
+requests-mock = "==1.5.2"
+django-webtest = "==1.9.3"
+freezegun = "==0.3.11"
```

Comment:
So apparently passing `--keep-outdated` to pipenv does nothing, and attempting to add this one dinky dependency caused it to upgrade every other dependency, which caused tons of warnings and test failures and what have you. The only solution appears to be pinning all the versions in `Pipfile`, which is something I wish pipenv did by default.